src/components/institution_table/InstitutionTable.js

```diff
@@ -74,14 +74,13 @@ export default function InstitutionTable(props) {
       div({style: Styles.TABLE.CONTAINER}, [
         div({style: Styles.TABLE.HEADER_ROW}, [tableHeaderTemplate]),
         filteredList.slice((currentPage - 1) * tableSize, (currentPage * tableSize)).map((inst, index) => {
-          let signingOfficials = '';
-          isNil(inst.signingOfficials) ? signingOfficials = '' : inst.signingOfficials.forEach((user, i) => {
-            signingOfficials = signingOfficials.concat(user.displayName, ' (', user.email, ')');
-            //if there are multiple SOs in the list and this is not the last one, add comma
-            if (inst.signingOfficials.length > 1 && i < (inst.signingOfficials.length - 1)) {
-              signingOfficials = signingOfficials.concat(', ');
-            }
-          });
+          let signingOfficialsList = [];
+          if (!isNil(inst.signingOfficials)) {
+            inst.signingOfficials.forEach((user, i) => {
+              signingOfficialsList.push(user.displayName.concat(' (', user.email, ')'));
```

Comment:
A cleaner way to write this string is to use template strings. So instead of a concat statement, you can type ``` `${user.displayName} (${user.email})` ```